en/docs/develop/writing-a-custom-local-authenticator.md

```diff
@@ -410,20 +449,18 @@ Let's begin.
 
 13. Under **Local & Outbound Authentication Configuration**, select **Local
     authentication** check box.
-    ![Local authentication check box](../assets/img/using-wso2-identity-server/local-authentication-check-box.jpeg) 
+    ![Local authentication check box](../assets/img/using-wso2-identity-server/local-authentication-check-box.png) 
 
     !!! note
-        On the corresponding drop-down list, you can see, `BasicCustom` which is the display name of the custom authenticator that was written. From the above step, you can make sure your custom authenticator is there and ready for use.    
+        On the corresponding drop-down list, you can see, `sample-local-authenticator` which is the display name of the custom authenticator that was written. Thus, you can make sure your custom authenticator is there and ready for use.    
```

Comment:
```suggestion
        On the corresponding drop-down list, you can see `sample-local-authenticator`, which is the display name of the custom authenticator that was written. Thus, you can make sure your custom authenticator is there and ready for use.    
```

Reply:
"custom authenticator is deployed correctly and ready for use. "